app/models/service.rb

```diff
@@ -61,6 +61,8 @@ def self.provider_specific_options(omniauth)
     case omniauth['provider'].to_sym
       when :'37signals'
         { user_id: omniauth['extra']['accounts'][0]['id'], name: omniauth['info']['name'] }
+      when :'lifx'
+        { name: 'LIFX User' }
```

Comment:
I suppose the the LIFX API does not return any useful information about the user?